tests/common.go

```diff
@@ -2816,7 +2816,7 @@ func CreateSourceAndDestClusters(orgID string, cloudName string, uid string, ctx
 		}
 		return "", false, nil
 	}
-	_, err = task.DoRetryWithTimeout(sourceClusterStatus, 2*time.Minute, 10*time.Second)
+	_, err = task.DoRetryWithTimeout(sourceClusterStatus, 4*time.Minute, 10*time.Second)
```

Comment:
We should parametrize these timeout in const

Reply:
Sure. Will parametrize these

Reply:
done